.github/workflows/run.yml

```diff
@@ -16,18 +16,13 @@
             python-version: "3.10"
 
         - name: Install dependencies
-          run: |
-            # $CONDA is an environment variable pointing to the root of the miniconda directory
-            $CONDA/bin/conda env update --file environment.yml --name base
+          run: $CONDA/bin/conda env update --file environment.yml --name base
 
         - name: Add $PYTHONPATH
-          run: |
-            echo "PYTHONPATH=/home/runner/work/scheduler/scheduler" >> $GITHUB_ENV
+          run: echo "PYTHONPATH=/home/runner/work/scheduler/scheduler" >> $GITHUB_ENV
```

Comment:
I guess that fixed it.

Reply:
Seems like such a trivial thing... I'm surprised that that worked...